pom.xml

```diff
@@ -22,36 +26,44 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!--Dependencies-->
-    <raml-module-builder.version>35.3.0</raml-module-builder.version>
+    <raml-module-builder.version>35.4.0</raml-module-builder.version>
+    <spring.version>6.2.3</spring.version>
     <jaxb-api.version>2.3.3</jaxb-api.version>
-    <vertx.version>4.5.10</vertx.version>
-    <rest-assured.version>5.5.0</rest-assured.version>
-    <sshd.version>2.13.2</sshd.version>
-    <spring.version>6.1.14</spring.version>
-    <folio-di-support.version>2.1.0</folio-di-support.version>
-    <assertj-core.version>3.26.3</assertj-core.version>
+    <vertx.version>4.5.13</vertx.version>
+    <rest-assured.version>5.5.1</rest-assured.version>
+    <sshd.version>2.15.0</sshd.version>
+    <assertj-core.version>3.27.3</assertj-core.version>
     <aspectj.version>1.9.22.1</aspectj.version>
-    <log4j.version>2.24.1</log4j.version>
+    <log4j.version>2.24.3</log4j.version>
+    <streamex.version>0.8.3</streamex.version>
+    <caffeine.version>3.2.0</caffeine.version>
 
     <!--Folio dependencies properties-->
     <folio-module-descriptor-validator.version>1.0.0</folio-module-descriptor-validator.version>
     <mod-configuration-client.version>5.11.0</mod-configuration-client.version>
+    <mod-di-converter-storage-client.version>2.3.1</mod-di-converter-storage-client.version>
+    <folio-di-support.version>3.0.0</folio-di-support.version>
+    <folio-kafka-wrapper.version>3.2.0</folio-kafka-wrapper.version>
+    <data-import-processing-core.version>4.4.0-SNAPSHOT</data-import-processing-core.version>
```

Comment:
we will change to release version when data-import-processing-core 4.4.0 would be released